src/packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -414,6 +414,14 @@ export function getPrismaClient(config: GetPrismaClientOptions): any {
           activeProvider: config.activeProvider,
         }
 
+        // Append the mongodb experimental flag if the provider is mongodb
+        if (config.activeProvider === 'mongodb') {
+          const enableExperimental = this._engineConfig.enableExperimental
+            ? this._engineConfig.enableExperimental.concat('mongoDb')
+            : ['mongoDb']
+          this._engineConfig.enableExperimental = enableExperimental
+        }
+
```

Comment:
Any place I should test this? I'm a bit unsure about how to test things like this with our test setup at the moment.  

Also `npm run test` fails on my machine currently for some unrelated issues.